extension-manifest-v3/scripts/build.js

```diff
@@ -67,6 +67,14 @@ if (argv.target === 'safari') {
   options.assets.push('background/safari');
 }
 
+if (argv.staging) {
+  execSync('npm run download.staging', { stdio: 'inherit' });
```

Comment:
Import and run download explicitly, not as an external command

Reply:
I'm fine with having calling those as javascript function but lets address this in separate PR.